### PR TITLE
[DOC] Arrange Ingest APIs Pipeline TOC

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Arrange TOC

### Issues Resolved
NA

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/ingest-apis/index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ redirect_from:
 
 # Ingest APIs
 
-Before you index your data, OpenSearch's ingest APIs help transform your data by creating and managing ingest pipelines. Pipelines consist of **processors**, customizable tasks that run in the order they appear in the request body. The transformed data appears in your index after each of the processor completes.
+Ingest APIs help you efficently transform and process data for an index by creating pipelines for dat ingestion in OpenSearch. Ingest pipelines use **processors**, customizable tasks that run in the order that they appear in the request body (that is, the data sent to the server as part of the HTTP request to ingest data in an API). Once the data is transformed, processed, and indexed, it is stored in OpenSearch and ready for you to search.
```
Comment (cwillum) [inline _api-reference/ingest-apis/index.md:12]: As vale points out, typo: efficiently
"... creating pipelines for data ingestion ..." ?

**Review by cwillum — APPROVED** [review]
LGTM

File: _api-reference/ingest-apis/get-ingest.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ---
 layout: default
 title: Get ingest pipeline
-parent: Ingest APIs
+parent: Pipelines
+grand_parent: Ingest APIs
 nav_order: 10
 redirect_from:
   - /opensearch/rest-api/ingest-apis/get-ingest/
+  - /api-reference/ingest-apis/index/
```
Comment (hdhalter) [inline _api-reference/ingest-apis/get-ingest.md:9]:
```suggestion
  - //api-reference/ingest-apis/get-ingest/
```

File: _api-reference/ingest-apis/get-ingest.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 ---
 
 ## Get ingest pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/get-ingest.md:12]: This should be H1 (just one #)

File: _api-reference/ingest-apis/create-update-ingest.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
 ---
 layout: default
 title: Create or update ingest pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/create-update-ingest.md:3]:
```suggestion
title: Create and update ingest pipeline
```

File: _api-reference/ingest-apis/create-update-ingest.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
-parent: Ingest APIs
+parent: Pipelines
+grand_parent: Ingest APIs
 nav_order: 11
 redirect_from:
   - /opensearch/rest-api/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/create-update-ingest/
 ---
 
 # Create and update a pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/create-update-ingest.md:12]:
```suggestion
# Create and update ingest pipeline
```

File: _api-reference/ingest-apis/simulate-ingest.md [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
-nav_order: 13
+parent: Pipelines
+grand_parent: Ingest APIs
+nav_order: 12
 redirect_from:
   - /opensearch/rest-api/ingest-apis/simulate-ingest/
+  - /api-reference/ingest-apis/simulate-ingest/
 ---
 
 # Simulate a pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/simulate-ingest.md:12]:
```suggestion
# Simulate an ingest pipeline
```

File: _api-reference/ingest-apis/delete-ingest.md [hunk 7]
```diff
@@ -1,10 +1,12 @@
 ---
 layout: default
 title: Delete a pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/delete-ingest.md:3]:
```suggestion
title: Delete ingest pipeline
```

File: _api-reference/ingest-apis/delete-ingest.md [hunk 8]
```diff
@@ -1,10 +1,12 @@
...
-nav_order: 14
+parent: Pipelines
+grand_parent: Ingest APIs
+nav_order: 13
 redirect_from:
   - /opensearch/rest-api/ingest-apis/delete-ingest/
+  - /api-reference/ingest-apis/delete-ingest/
 ---
 
 # Delete a pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/delete-ingest.md:12]:
```suggestion
# Delete ingest pipeline
```

File: _api-reference/ingest-apis/simulate-ingest.md [hunk 9]
```diff
@@ -1,15 +1,17 @@
 ---
 layout: default
 title: Simulate an ingest pipeline
```
Comment (hdhalter) [inline _api-reference/ingest-apis/simulate-ingest.md:3]:
```suggestion
title: Simulate ingest pipeline
```

File: _api-reference/ingest-apis/simulate-ingest.md (hunk 6 above)
Comment (hdhalter) [inline _api-reference/ingest-apis/simulate-ingest.md:12]:
```suggestion
# Simulate ingest pipeline
```

**Review by hdhalter — COMMENTED** [review]
Looks good! Just made some comments on the titles, as they are not parallel in structure, a formatting thing, and an incorrect redirect.

**Comment by vagimeli** [comment]
Addressed navigation while closing content gaps on this subject area.

**Pull request closed without merging**